smoketest.bash

```diff
@@ -205,6 +208,22 @@ if [ "${USE_PROXY}" = "true" ]; then
     createProxyCfgVolume
 fi
 
+createProxyCertsVolume() {
+    "${container_engine}" volume create auth_proxy_certs
+    "${container_engine}" container create --name proxy_certs_helper -v auth_proxy_certs:/certs busybox
+    if [ -f "${DIR}/compose/auth_certs/certificate.pem" ] && [ -f "${DIR}/compose/auth_certs/private.key" ]; then
+        chmod 644 "${DIR}/compose/auth_certs/private.key"
+        "${container_engine}" cp "${DIR}/compose/auth_certs/certificate.pem" proxy_certs_helper:/certs/certificate.pem
+        "${container_engine}" cp "${DIR}/compose/auth_certs/private.key" proxy_certs_helper:/certs/private.key
+    else
+        echo "Unable to find a certificate and key to allow oauth2_proxy to enable TLS connections"
```

Comment:
Just to make it easier for someone in the future, maybe this should explicitly ask "did you run auth_certs/generate.sh?"